File: c ass/linkedlist.c
#include <stdio.h>
#include<malloc.h>
typedef struct node
{
	int data;
	struct node*next;
}nin;
nin *head,*first,*second,*op;
void  display();

int main(void) {
	
	head=(nin*) malloc(sizeof(nin));
	first=(nin*) malloc(sizeof(nin));
	second=(nin*) malloc(sizeof(nin));
	head->data=10;
	head->next=first;
	first->data=20;
	first->next=second;
	second->data=30;
	second->next=NULL;
	display();
	
}
	void display()
	{
		
		op=head;
		while(op!=NULL)
		{
			printf("%d\n",op->data);
			op=op->next;
		}
	}
	

File: JAVA/linkedlist.java
package rtt;

 class Dent {
	int data;
	Dent next;
	Dent(int a)
	{
		data=a;
		next=null;
	}
}
 public class X
{
	void display(Dent temp)
	{
		while(temp!=null)
		{
			System.out.println(temp.data);
			temp=temp.next;
		}
	}
static Dent head,first,second,temp;
public static void main(String args[])
{
	head=new Dent(10);
	first=new Dent(20);
	second=new Dent(30);
	head.next=first;
	first.next=second;
	X c=new X();
	c.display(head);
	
}

}
